### ISSUE TITLE
Short read performance

### ISSUE THREAD

**Issue by jarrettrios** [readwrite112/AGAThA#3]
How does AGAThA perform for the alignment of short reads? Meaning 250 - 500 bps, rather than the Kbp's range.

**Comment by readwrite112** [comment]
Are 250-500 bps short reads referring to next generation sequencing results? 
While I have not directly tested this out, I expect the amount of speedup to be relatively smaller since a large portion of this work's motivation and speedup comes from the large amount of workload imbalance between alignments.

**Comment by jarrettrios** [comment]
Yes, I was referring to NGS results. I was looking at GASAL2, but it looked like it receives almost no support, which makes me nervous to use it as a part of our pipeline. I noticed that this both more recent and you/your team are actively supporting it! Even if the speed up compared to GASAL2, is negligible, it would still be a huge jump compared to our current pipeline. I wish github had a forum option, so I could ask these questions without making forum posts. Thank you for taking the time to respond!
